tap_shopify/streams/base.py

```diff
@@ -45,7 +49,8 @@ def retry_after_wait_gen(**kwargs):
 
 def shopify_error_handling(fnc):
     @backoff.on_exception(backoff.expo,
-                          pyactiveresource.connection.ServerError,
+                          (pyactiveresource.connection.ServerError,
+                           simplejson.scanner.JSONDecodeError),
```

Comment:
Are they naively decoding json in the library? Is that why we need to catch this?

Reply:
Yes, they catch the error and bubble up.